### ISSUE TITLE
403 PerimeterX Error

### ISSUE THREAD

**Issue by mhaas2-tgm** [matthew1232/stockx-api#19]
When I try to run this Code to get Products from a Keyword:
```
stockX.newSearchProducts(productName, {
                limit: 1
            })
            .then(products => {
                resp.send("Result: "+products[0].name)
                return null
            })
            .catch(err => {
                resp.send("Failed; "+err)
            });
```
It always returns a weird error:
```
StatusCodeError: 403 - "\n\n\n \n \n \n \n \n \n \n\n\n\n
\n
\n
\n \"Logo\"/\n
\n
\n
\n
\n
Please verify you are a human
\n
\n
\n
\n
\n\n
\n
\n
\n Access to this page has been denied because we believe you are using automation tools to browse the\n website.\n

\n
\n This may happen as a result of the following:\n

\n
\n
\n Javascript is disabled or blocked by an extension (ad blockers for example)\n
\n
\n Your browser does not support cookies\n
\n
\n
\n Please make sure that Javascript and cookies are enabled on your browser and that you are not blocking\n them from loading.\n

\n
\n Reference ID: #21dfa960-e799-11ea-b8d1-35b830d5ef89\n

\n
\n
\n
\n
\n
\n Powered by\n PerimeterX\n , Inc.\n

\n
\n
\n
\n\n\n\n\n\n\n\n"
```

**Comment by arponp** [comment]
I am having the same error

**Comment by sayansisodiya** [comment]
I don't know if this is a permanent solution, but I was able to get around this error by manually logging out of and back into stockx.com through a browser. Once I verified that I was human, the API functionality worked again.

**Comment by matthew1232** [comment]
Should be fixed in update 1.1.0, please download the new version and try again.
